--- checks.py	2024-03-19 03:56:34.547652655 +0000
+++ checks_new.py	2024-03-19 04:11:36.630926035 +0000
@@ -526,8 +526,6 @@
 def check_imshow(warn=False):
     """Check if environment supports image displays."""
     try:
-        if LINUX:
-            assert "DISPLAY" in os.environ and not is_docker() and not is_colab() and not is_kaggle()
         cv2.imshow("test", np.zeros((8, 8, 3), dtype=np.uint8))  # show a small 8-pixel image
         cv2.waitKey(1)
         cv2.destroyAllWindows()
