--- autobackend.py	2024-03-19 03:56:34.543652589 +0000
+++ autobackend-new.py	2024-03-19 04:16:40.268140420 +0000
@@ -211,7 +211,7 @@
             LOGGER.info(f"Using OpenVINO {inference_mode} mode for batch-size={batch_size} inference...")
             ov_compiled_model = core.compile_model(
                 ov_model,
-                device_name="AUTO",  # AUTO selects best available device, do not modify
+                device_name="GPU",  # AUTO selects best available device, do not modify
                 config={"PERFORMANCE_HINT": inference_mode},
             )
             input_name = ov_compiled_model.input().get_any_name()
